components/combobox/combobox.jsx

```diff
@@ -1054,9 +1109,15 @@ class Combobox extends React.Component {
 							}}
 							placeholder={labels.placeholderReadOnly}
 							readOnly
+							required={props.required}
 							role="textbox"
 							value={value}
 						/>
+						{props.errorText && (
+							<div id={this.getErrorId()} className="slds-form-element__help">
+								{props.errorText}
+							</div>
+						)}
 						{this.getDialog({
```

Comment:
When `menuPosition` is `relative` which means the menu's CSS is `position: relative`, the menu needs to be placed after the input.

<img width="545" alt="screen shot 2018-03-13 at 9 30 28 pm" src="https://user-images.githubusercontent.com/1290832/37382038-09fee01e-2706-11e8-9795-ed96ea872258.png">